dashboard/src/components/Header/ContextSelector.tsx

```diff
@@ -178,7 +179,15 @@ function ContextSelector() {
                   </CdsModalActions>
                 </form>
               </CdsModal>
-              <CdsButton status="inverse" size="sm" action="flat" onClick={openNewNSModal}>
+              <CdsButton
+                disabled={!canCreateNS}
+                title={canCreateNS ? "" : "missing permissions"}
```

Comment:
We may as well include a useful title when the user can create a NS? "Create a new namespace in the current cluster" or similar? Also, I think it'd be worth at least a phrase for the missing permissions, perhaps? "You don't have permission to create namespaces on the cluster." Fine either way, just a thought.